ihontaryk/home_work_2.py

```diff
@@ -0,0 +1,39 @@
+""" Functions for Home work 2"""
+
+
+def fizz_buzz(number):
+    """fizz_buzz function"""
+    if type(number) is not int:
+        raise TypeError
+    if number == 0:
+        raise ValueError
+    if number % 15 == 0:
+        result = "FizzBuzz"
+    elif number % 3 == 0:
+        result = "Fizz"
+    elif number % 5 == 0:
+        result = "Buzz"
+    else:
+        result = str(number)
+
+    return result
```

Comment:
leave some space in code, use newlines to separate parts of code one from another